spinalcordtoolbox/types.py

```diff
@@ -209,10 +209,10 @@ def __init__(self, points_x=None, points_y=None, points_z=None, deriv_x=None, de
             self.derivatives = centerline_file['derivatives']
 
             if 'disks_levels' in centerline_file:
-                self.disks_levels = centerline_file['disks_levels'].tolist()
+                self.discs_levels = centerline_file['disks_levels'].tolist()
```

Comment:
That's it exactly. I considered writing some backward compatibility logic, to start emitting `disc_levels`, while accepting both `disc_levels` and `disk_levels`, but that would break forward compatibility.

I thought I added some warnings about the old spelling in code comments, but maybe that was on a different branch! I'll add some.

Reply:
Added in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4075/commits/eacee6bdf690e339d6ce1fc2a0b4655483f137ad.